src/zenml/cli/utils.py

```diff
@@ -2649,3 +2667,17 @@ def verify_mlstacks_prerequisites_installation() -> None:
         error(NOT_INSTALLED_MESSAGE)
     except subprocess.CalledProcessError:
         error(TERRAFORM_NOT_INSTALLED_MESSAGE)
+
+
+def is_jupyter_installed() -> bool:
+    """Checks if Jupyter notebook is installed.
+
+    Returns:
+        bool: True if Jupyter notebook is installed, False otherwise.
+    """
+    try:
+        import notebook
```

Comment:
Just a small nitpick here.

While this is much simpler, it might be more performant and safer to use the same functionality that we used for [our integrations to check](https://github.com/zenml-io/zenml/blob/cb57213189ae8a30591fdcfbd294d77201dbd856/src/zenml/integrations/integration.py#L64) whether the `notebook` package is installed, instead of importing the whole package.

Again, for this use case, we can still leave this as is.